### DISCUSSION TITLE
ERROR 2026 (HY000): SSL connection error: SSL_CTX_set_default_verify_paths failed

### DISCUSSION THREAD

**Discussion by Brenneisen** [laravel/framework#41299] [Q&A] [answered]
- Laravel Version: 9.3.0
- PHP Version: 8.1.1
- Database Driver & Version: 8.0.27

### Description:

I get an error when running `php artisan migrate`:

```
The command "mysql  --user="${:LARAVEL_LOAD_USER}" --password="${:LARAVEL_LOAD_PASSWORD}" --host="${:LARAVEL_LOAD_HOST}" --port="${:LARAVEL_LOAD_PORT}" --ssl-ca="${:LARAVEL_LOAD_SSL_CA}" --database="${:LARAVEL_LOAD_DATABASE}" < "${:LARAVEL_LOAD_PATH}"" failed.

Exit Code: 1(General error)

Output:
================

Error Output:
================
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 2026 (HY000): SSL connection error: SSL_CTX_set_default_verify_paths failed
```

### Steps To Reproduce:

Run `php artisan migrate`.

**Comment by Brenneisen** [comment]
@driesvints Why did you convert the issue into a discussion? With version `9.2.0` the migrations still work. With `9.3.0` our integration tests throw an error when migrating because of an SSL connection error. For me it seems to be due to the new config `--ssl-ca="${:LARAVEL_LOAD_SSL_CA}"`. What needs to be changed to fix this breaking change?

**Reply by driesvints** [reply]
@Brenneisen there wasn't much info here so I thought this was a support questions. It would seem weird to me that this wouldn't work for you specifically. If you said that it worked on 9.2.0 I would probably have kept it open.

**Reply by driesvints** [reply]
I can't re-open it. Can you please submit it again with the info from above?

**Reply by Brenneisen** [reply]
Done

**Reply by justlyall** [reply]
Yup that --ssl-ca="${:LARAVEL_LOAD_SSL_CA}" is giving me problems as well

**Reply by driesvints** [reply, marked as answer]
v9.3.1 was released today that addresses this.
